app/assets/stylesheets/content/ui_select.sass

```diff
@@ -1,6 +1,7 @@
 // keep this rule if you want ui-select drag-n-drop to work properly
-.ui-select-match .ui-select-match-item *
-  pointer-events: none
+.ui-select-match .ui-select-match-item
+  span, span *
+    pointer-events: none
```

Comment:
and the span is there to distingush with the anchor that actually needs the pointer event to be enabled because otherwise click does not work. year 2015, disabling javascript events via CSS